packages/utils/test/crypto/ec-utils.test.ts

```diff
@@ -27,7 +27,6 @@ describe('Utils/EcUtils', () => {
   describe('getAddressFromPrivateKey', () => {
     it('can get Address From PrivateKey', () => {
       const identity = getAddressFromPrivateKey(rawId.privateKey);
-      // 'getAddressFromPrivateKey() error'
       expect(identity).toBe(rawId.address);
```

Comment:
Those comments were messages displayed in case of a failure when we were still using mocha instead of jest, they were placed as the second argument in the `expect`, see https://github.com/RequestNetwork/requestNetwork/pull/283. They can be removed, at least for happy-path tests, they can be confusing.

Reply:
I removed them for failure tests too, see https://github.com/RequestNetwork/requestNetwork/pull/1239#discussion_r1392407982